src/components/settings/SettingsAccessTab/ActionsTableCell.jsx

```diff
@@ -25,7 +30,7 @@ const ActionsTableCell = ({ row, onDeactivateLink, enterpriseUUID }) => {
       return;
     }
     const addToClipboard = async () => {
-      const inviteURL = `${getConfig().ENTERPRISE_LEARNER_PORTAL_URL}/invite/${inviteKeyUUID}`;
+      const inviteURL = `${getConfig().ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/invite/${inviteKeyUUID}`;
```

Comment:
nit: i wonder if it's worth exposing a lil' helper function (e.g., `getUniversalLinkUrl` or similar) to abstract the generation of this URL to one place vs. the two we have today.